user_tools/src/spark_rapids_pytools/rapids/qualification.py

```diff
@@ -192,6 +193,51 @@ class Qualification(RapidsJarTool):
     """
     name = 'qualification'
 
+    def __calculate_spark_settings(self, worker_info: NodeHWInfo) -> dict:
```

Comment:
Would there be a way to move this function into a utility library so that it's shared between qualification and bootstrap?